roadmap.md

```diff
@@ -0,0 +1,35 @@
+## Roadmap
+
+The CloudEvents Roadmap.
+
+*January*
+
+* Establish governance, contributing guidelines and initial stakeholders.
+* Agree on design goals for CloudEvents V.1.
+* Draft educational materials on how to become a stakeholder.
+* Draft educational materials on use-cases.
+* Iterate on version 0.1 of the specification.
+
+*February*
+
+* Draft educational materials on use-cases.
+* Collaborate on libraries and supporting tools to use CloudEvents and
+integrate it with the ecosystem.
+* Finalize version 0.1 of the specification by the end of February, 2018.
+
+*March*
+
+* Collaborate on libraries and supporting tools to use CloudEvents and
+integrate it with the ecosystem.
+* Draft documentation and user guides.
+* Build out promotional materials (website, logos, etc.)
```

Comment:
minor nit but I think the period goes outside the parens.  Below too.

Reply:
lol.  Period is for the abbreviation.  It also needs a period after the sentence though.